### ISSUE TITLE
[object Object] Internal MythX Errors

### ISSUE THREAD

**Issue by nbanmp** [ConsenSys/truffle-security#97]
Because of how we are concatenating error messages, they are sometimes not displayed correctly.

e.g.
```
Internal MythX errors encountered:
SafeMath: [object Object]
```

**Comment by tagomaru** [comment]
@nbanmp 
I have not been able to reproduce it. Could you add the following code behind [here](https://github.com/ConsenSys/truffle-security/blob/master/helpers.js#L359) to see the contents of that object?
I will check it later.

===
config.logger.log(JSON.stringify(err, null, 2));
config.logger.log(typeof err);
===

Thx!

**Comment by rocky** [comment]
I suspect this is related to #109 which may explain why it is seen for some and not other and at different places. 

I have observed that once the initial 10 minute timeout period for the first access token occurs and a refresh successfully occurs, you have a day before the next (refreshed) access token times out.

**Comment by rocky** [comment]
Fixed by #110
